frontend/src/Component/PodcastForm.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {ChangeEvent, FormEvent, useState} from 'react';
+import axios from 'axios';
+
+function PodcastForm() {
+    const [podcast, setPodcast] = useState('');
+
+    const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        try {
```

Comment:
Dont't use try catch here... catch errors directly in the axios call. **.catch(() => {})